cnd/src/http_api/routes/rfc003/mod.rs

```diff
@@ -51,15 +57,18 @@ pub fn get_swap<T: MetadataStore, S: StateStore>(
         .map_err(into_rejection)
 }
 
+// TODO: Remove two arguments here (they are in the dependencies struct)
```

Comment:
Uff, I left this here :sweat_smile: 

Do you mind removing it, otherwise it is gonna open an issue ...

Reply:
We have a million clippy issues and I only have 10 minuets until I'm off to get Milla.